gdcdictionary/schemas/gdc_release.yaml

```diff
@@ -0,0 +1,80 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "gdc_release"
+title: GDC Release 
+type: object
+namespace: http://gdc.nci.nih.gov
+category: administrative 
+program: '*'
+project: '*'
```

Comment:
Can we restrict these?

Reply:
Every schema has those same values for program and project. I'm guessing program would be "GDC", but would project be "MISC" or "GDC-MISC".  Any ideas @madopal ?

Reply:
My guess is that project would be "MISC". This is fine for now, we can test it on our cluster.

Reply:
Yeah, it says `GDC-MISC` in the ticket https://jira.opensciencedatacloud.org/browse/TT-511
Don't forget to restrict it before the dictionary release though

Reply:
Meaning, `program=GDC`, `project=MISC`

Reply:
Project has been restricted to GDC-MISC.